examples/deadline/Local-Zone/README.md

```diff
@@ -0,0 +1,39 @@
+# RFDK Sample Application - Local Zones
```

Comment:
This example does a great job of showing how to deploy the Workers to a local zone - great work!

I think one thing that might be missing though is an explanation of why this is important/useful. RFDK users want this so they can have a low-latency connection between their AWS infrastructure and their infrastructure hosted outside AWS. It might help to spend some time at the beginning of this README setting the stage here. It may also help to add some guidance, links, and next steps to point users in the right direction for how they'd take this exmaple and proceed to the next step of connecting their infrastructure outside of AWS.

One good resource we could link to is our developer guide documentation on [connecting to a RFDK render farm](https://docs.aws.amazon.com/rfdk/latest/guide/connecting-to-render-farm.html)

Reply:
Thanks for adding documentation for this in the top-level README.

As discussed offline, let's add a "next steps" section to the Python/TS READMEs that list the steps required to connect an on-premise networked file-system to the Workers. Even if we don't have all of the steps completely detailed, for now having a checklist would help guide readers to research and complete them. These are the steps that I foresee here:

1.  Follow the [connecting to your render farm](https://docs.aws.amazon.com/rfdk/latest/guide/connecting-to-render-farm.html) dev guide page to modify the `NetworkTier` with the the VPN connection.
2. Ensure there is a route from the worker subnet to the on-prem file-system
3. Ensure the VPN endpoint's security group has ingress rules to allow NFS port traffic from the Workers
4. Add user-data to mount the NFS on the compute tier
5. (optional) set up path mapping rules in Deadline